MODELS/utils.models.js

```diff
@@ -0,0 +1,22 @@
+const db = require('../db/connection');
+
+exports.columnValidate = async (columnName) => {
+  const {rows: [{exists}]} = await db.query(`
+  SELECT EXISTS 
+  (SELECT 1 FROM information_schema.columns 
+    WHERE table_name='articles' 
+    AND column_name=$1);`, [columnName]);
+  return exists;
+};
```

Comment:
I like your thinking here but we should be minimising the number of db requests to optimise performance if your project expands. Also, as you are using pg-format in your model this isn't needed as %I handles the column name like $1 does here!